ydb/docs/presets.yaml

```diff
@@ -64,3 +64,24 @@ default:
   ydb-finebi: false
   ydb-superset: false
   ydb-datalens: false
+
+#=== YDB default settings ===#
+
+  ydb-ports:
+    grpcs: 2135
+    grpc: 2136
+    interconnect: 19001
+    https: 8765
+    tracing: 4317
+
+#=== JDBC default settings ===#
+
+  third-party-ports:
+    fq: 2130
```

Comment:
This one is not third-party, it's fq-connector from https://github.com/ydb-platform/fq-connector-go 

Let's move it to ydb-ports and rename to fq-connector